tpgtools/overrides/compute/beta/service_attachment.yaml

```diff
@@ -0,0 +1,8 @@
+- type: EXCLUDE
+  field: region
```

Comment:
Ah, weird- the base DCL resource includes a (presumably output-only) `region` field but also a `location` field which maps to the `region` field in the API?

Reply:
Yes, this is the way KCC requested we name the fields. `location` corresponds to the URL parameter.